File: week7/methodoverload.cpp
#include<iostream>
using namespace std;

class addition{
   public:
       int add(float a, float b)
       {
           return a+b;
       }
       int add(float a, float b, float c)
       {
           return a+b+c;
       }
};
int main()
{
   addition obj;
   cout << "The addition of two numbers is "<< obj.add(4,5) << endl;
   cout << "The addition of three numbers is "<< obj.add(4,5,1) << endl;
   return 0;
}

File: week7/methodoverload.java
import java.util.Scanner;

public class methodoverload {
   public int add(int a, int b){
       return a+b;
   }
   public int add(int a, int b, int c){
       return a+b+c;
   }
   public static void main(String[] args) {
       methodoverload obj = new methodoverload();
       System.out.println("The addition of two numbers is "+obj.add(4,5));
       System.out.println("The addition of three numbers is "+obj.add(4,5,1));
   }
}
